web/content/index.adoc.erb

```diff
@@ -41,7 +41,11 @@ If you are currently using one of the unsupported earlier releases, we recommend
 
 The future version is built in nightly:
 
-* link:/release/nightly/[Nightly]
+* link:/release/nightly/[Nightly] 
+** link:/release/nightly/index-foreman-el.html[Foreman on Enterprise Linux (EL)]
+** link:/release/nightly/index-foreman-deb.html[Foreman on Debian/Ubuntu]
+** link:/release/nightly/index-katello.html[Foreman with the Katello plugin on Enterprise Linux (EL)]
+
 
 <% if (release = releases_in_state('RC').first) %>
```

Comment:
Don't forget the RCs :)

Reply:
I added the list here as well but was unable to figure out how to test whether it works.

Reply:
We'll see in 2 weeks when we branch Foreman 3.16? :)